pkg/compute/init.go

```diff
@@ -68,6 +71,7 @@ func NewInitCommand(parent common.Registerer, globals *config.Data) *InitCommand
 	c.CmdClause.Flag("description", "Description of the package").Short('d').StringVar(&c.description)
 	c.CmdClause.Flag("author", "Author of the package").Short('a').StringVar(&c.author)
 	c.CmdClause.Flag("from", "Git repository containing package template").Short('f').StringVar(&c.from)
+	c.CmdClause.Flag("from-branch", "Git branch name to clone from package template repository").Hidden().StringVar(&c.fromBranch)
```

Comment:
This feels like it overloads `--from`. What about `--branch` or `--remote-branch`?

Reply:
Resolved in https://github.com/fastly/cli/pull/14/commits/541d1419a04dcd60e00d52a7294036342380db06